### ISSUE TITLE
This plugin slows down Homebridge

### ISSUE THREAD

**Issue by higgyforever** [apatsufas/homebridge-tapo-p100#23]
**Describe The Bug:**
Homebridge log shows the following message:
_[homebridge-tapo] This plugin slows down Homebridge. The read handler for the characteristic 'On' was slow to respond! See https://git.io/JtMGR for more info._

**To Reproduce:**
Hard to say how to reproduce this. I got 3 plugins running. "Hombridge Hue", "Homebridge UI" and this plugin. Homebridge is running on a raspberry running Raspbian GNU/Linux 10 (buster). I am running node v16.13.1. Homebridge is running in version v1.3.8.
homebridge ui v4.41.4
Homebridge Hue v0.13.30
I got 4 tapo switches, two of then are not online at the moment but configured.

**Expected behavior:**
This message should not appear. I believe the plugin should return the call earlier and run the status change to the device (e.g. switch on/off) in the background.

**Logs:**
Dec 20 20:01:17 HOSTNAME homebridge[21935]: [20.12.2021, 20:01:17] Homebridge v1.3.8 (HAP v0.9.7) (Homebridge) is running on port 50001.
Dec 20 20:01:19 HOSTNAME homebridge[21935]: [20.12.2021, 20:01:19] [Hue] Wohnzimmer: set homekit supported transition configuration to {"configuration":[{"iid":11,"characteristic":1},{"iid":13,"characteristic":2}]}
Dec 20 20:01:19 HOSTNAME homebridge[21935]: [20.12.2021, 20:01:19] [Hue] Stehlampe: set homekit supported transition configuration to {"configuration":[{"iid":11,"characteristic":1},{"iid":13,"characteristic":2}]}
Dec 20 20:01:19 HOSTNAME homebridge[21935]: [20.12.2021, 20:01:19] [Tapo Smart Platform] 111 Error: connect EHOSTUNREACH 192.168.188.79:80
Dec 20 20:01:19 HOSTNAME homebridge[21935]: [20.12.2021, 20:01:19] [Tapo Smart Platform] 52 - Get Device Info failed
Dec 20 20:01:19 HOSTNAME homebridge[21935]: [20.12.2021, 20:01:19] [Tapo Smart Platform] 111 Error: connect EHOSTUNREACH 192.168.188.78:80
Dec 20 20:01:19 HOSTNAME homebridge[21935]: [20.12.2021, 20:01:19] [Tapo Smart Platform] 52 - Get Device Info failed
Dec 20 20:01:25 HOSTNAME homebridge[21935]: #033[0;37m[20.12.2021, 20:01:24] #033[0m#033[0;36m[Homebridge UI]#033[0m #033[0;33mHomebridge Config UI X v4.41.4 is listening on :: port 8080#033[0m
Dec 20 20:01:26 HOSTNAME homebridge[21935]: [20.12.2021, 20:01:26] [Hue] Corner Light: homekit brightness changed from 70% to 70%
Dec 20 20:02:00 HOSTNAME homebridge[21935]: [20.12.2021, 20:02:00] [Tapo Smart Platform] 371 Error: timeout of 2000ms exceeded
Dec 20 20:02:03 HOSTNAME homebridge[21935]: [20.12.2021, 20:02:03] [Tapo Smart Platform] 371 Error: timeout of 2000ms exceeded
Dec 20 20:02:04 HOSTNAME homebridge[21935]: [20.12.2021, 20:02:04] [homebridge-tapo] This plugin slows down Homebridge. The read handler for the characteristic 'On' was slow to respond! See https://git.io/JtMGR for more info.
Dec 20 20:02:10 HOSTNAME homebridge[21935]: [20.12.2021, 20:02:10] [homebridge-tapo] This plugin slows down Homebridge. The read handler for the characteristic 'On' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
Dec 20 20:07:34 HOSTNAME systemd[1]: Started Session 109 of user USER.
Dec 20 20:10:58 HOSTNAME homebridge[21935]: [20.12.2021, 20:10:58] [Hue] Corner Light: homekit brightness changed from 70% to 70%

Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
{
            "name": "Tapo Smart Platform",
            "username": "username",
            "password": "password",
            "devices": [
                {
                    "name": "TapoSwitch1",
                    "host": "tapo-smartplug-1",
                    "type": "Plug"
                },
                {
                    "name": "TapoSwitch2",
                    "host": "tapo-smartplug-2",
                    "type": "Plug"
                },
                {
                    "name": "TapoSwitch3",
                    "host": "tapo-smartplug-3",
                    "type": "Plug"
                },
                {
                    "name": "TapoSwitch4",
                    "host": "tapo-smartplug-4",
                    "type": "Plug"
                }
            ],
            "platform": "TapoP100"
        }
```

**Screenshots:**
No screenshots.

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 1.3.0
* **Node.js Version**: 16.13.1
* **NPM Version**: 8.3.0
* **Operating System**:  Raspbian GNU/Linux 10 (buster)

**Comment by apatsufas** [comment]
Should be fixed with 1.4.15

**Comment by higgyforever** [comment]
Thank you!
